packages/toolpad-components/src/Chart.tsx

```diff
@@ -172,11 +174,26 @@ function Chart({ data = [], height, sx }: ChartProps) {
           })}
         </ComposedChart>
       </ResponsiveContainer>
+      {loading ? (
```

Comment:
Would it make sense to use the Skeleton, like we do for images? https://github.com/mui/mui-toolpad/blob/5a1a5717e8aee85a428e55bfb18727309c60da89/packages/toolpad-components/src/Image.tsx#L69

Reply:
The chart components being used in the examples had a spinner like this, I'll see if that looks better and go with the best option.

Reply:
I think we can keep the spinner here - the chart already shows an outline of where the axes are going to be before it loads the data, and it would be a bit manual and probably unnecessary to make the skeleton rectangle fit inside those axes.
If anything changes when adding the MUI X charts I'll keep this in mind.